operators/src/source/gdal_source.rs

```diff
@@ -285,7 +287,7 @@ where
             .time_step
             .snap_relative(time_information.start_time, time_interval.start())
             .expect("is a valid time");
-        eprintln!("snapped start {:?}", snapped_start);
```

Comment:
is there a lint against "print" in the code? :dagger:

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#print_stderr

Reply:
we should use it